WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -122,14 +122,21 @@ def results_submitted(competition, results_submission, submitter_user)
     @competition = competition
     @results_submission = results_submission
     @submitter_user = submitter_user
-    file_name = "Results_#{competition.id}_#{Time.now.utc.iso8601}.json"
-    attachments[file_name] = results_submission.results_json_str
+    last_uploaded_json = @competition.uploaded_jsons.last
```

Comment:
Do you maybe need a `order` or `sort` here to ensure you get the last one? Right now, this might be working just because MySQL tends to return things in the order they were inserted, but I don't know if that's a guarantee...

Reply:
Oh right there is no guarantee!
I'll update this.